src/components/app/FooterLinks.js

```diff
@@ -3,14 +3,12 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // @flow
 
-import React, { PureComponent } from 'react';
```

Comment:
Why did this change?

Reply:
Mainly for consistency with other files. Not strictly necessary here as we don't use React's flow type in this file.

Reply:
That said, I see a lot of files haven't been changed automatically, I'm not sure why. Are the changed ones the same ones that use React flow types ? I'll dig in some more.

Reply:
I found the cause: the codemod automatically changed all `import React from 'react'` (hence the changes in all tests) but not the ones like this one where we also import `PureComponent` while destructuring. I manually changed the ones where I needed the new type, but not others. This one is an exception because I did it while merging master. I'll revert this change.

Not sure what we should use in the future to be consistent.